pkg/server/wire.go

```diff
@@ -197,6 +198,9 @@ var wireBasicSet = wire.NewSet(
 	dashboardimportservice.ProvideService,
 	wire.Bind(new(dashboardimport.Service), new(*dashboardimportservice.ImportDashboardService)),
 	plugindashboards.ProvideService,
+	intentapi.ProvideHTTPServer,
```

Comment:
I'd be fine to leave it as a follow-up (though it merits posting an issue, if we do), but i think we're better off starting a separate group of wire provider sets for the intent api HTTP server, rather than piling more into this main set.

Reply:
Sounds good to me - I'm personally in favour of doing this in a separate PR 🙂

Reply:
Okay, I've ended up adding a separate group in this PR. Let me know if that works for you.